commandlineparser/src/Options.cpp

```diff
@@ -1 +1,25 @@
-#include<iostream>
\ No newline at end of file
+#include"Options.h" 
+
+Options::Options(std::string short_command, std::string long_command,
+std::string short_description, std::string long_description)
+ :long_description(long_description),short_description(short_description)
+ {
+	 set_option_short_command(short_command);
+	 set_option_long_command(long_command);
+ }
+ 
+ Options::Options(std::string long_command):short_description(""),short_command(""),long_description("")
+ {  set_option_long_command(long_command);
+ 
+ }
+ 
+void Options::set_option_short_command(std::string short_command){if(!short_command.empty()){this->short_command='-'+short_command;}}
```

Comment:
follow one indentation type. Here you have used { in front of function other places its in line next to the function name

Reply:
is this empty check really needed ?